libs/blocks/modal/modal.js

```diff
@@ -65,9 +68,21 @@ async function getPathModal(path, dialog) {
   const block = createTag('a', { href: path });
   dialog.append(block);
 
+  // eslint-disable-next-line import/no-cycle
   const { default: getFragment } = await import('../fragment/fragment.js');
   await getFragment(block);
 }
+function sendViewportDimensionsToiFrame(source) {
+  const viewportWidth = Math.max(document.documentElement.clientWidth || 0, window.innerWidth || 0);
+  source.postMessage({ mobileMax: MOBILE_MAX, tabletMax: TABLET_MAX, viewportWidth }, '*');
+}
+
+export function sendViewportDimensionsOnRequest(messageInfo) {
+  if (messageInfo.data === 'viewportWidth') {
+    sendViewportDimensionsToiFrame(messageInfo.source);
+    window.addEventListener('resize', () => sendViewportDimensionsToiFrame(messageInfo.source));
```

Comment:
You need the debounce here

Reply:
The debounce has been added in the caller function.

Reply:
Yes that won't work because the resize event recieves a lot of calls - the caller function is irrelevant. You can test it by just checking how often the `sendViewportDimensionsToiFrame` function is called when you resize the window